src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -838,8 +850,11 @@ export const ShippingRoute: FC<ShippingProps> = props => {
                   ) => {
                     setAddressNeedsVerification(false)
                     setAddressVerifiedBy(verifiedBy)
-                    setAddress({ ...address, ...chosenAddress })
-                    saveAndContinue && finalizeFulfillment()
+                    setAddress(address => {
+                      return { ...address, ...chosenAddress }
+                    })
+                    setReadyToSaveVerifiedAddress(true)
+                    // saveAndContinue && finalizeFulfillment()
```

Comment:
Then I think we can remove the whole `saveAndContinue` argument with the proposed approach?